rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/BaseLanguageServer.java

```diff
@@ -313,6 +313,17 @@ public BaseWorkspaceService getWorkspaceService() {
             return lspWorkspaceService;
         }
 
+        @Override
+        public NotebookDocumentService getNotebookDocumentService() {
+            return null; // can be removed after lsp4j v0.16.0 is released (https://github.com/eclipse/lsp4j/issues/658)
+        }
+
+        @Override
+        public void setTrace(SetTraceParams params) {
```

Comment:
what are we ignoring here? perhaps add a little comment for future reference

Reply:
It's a new feature, we aren't using it yet, I'll add a comment (and issue) to say we should do this in the future.

Reply:
Added #177 for this.